### PR TITLE
Documentation for Météo France sensor

### PR DESCRIPTION
**Pull request by victorcerutti**
**Description:**
Météo France provides a convenient rain forecast that is pretty accurate.
This sensor gives the time to next rain forecasted.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>
home-assistant/home-assistant#17773

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_components/sensor.meteofrance.markdown [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: meteo_france
+    location_id: 75101
+    name: Paris 1er
+```
+
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/sensor.meteofrance.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/sensor.meteofrance.markdown [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+- 4 Heavy rain
+
+To add Météo-France to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+sensor:
+  - platform: meteo_france
+    location_id: 75101
+    name: Paris 1er
```
Comment (frenck) [inline source/_components/sensor.meteofrance.markdown]: We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps a user to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.

File: source/_components/sensor.meteofrance.markdown [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+The location ID is the INSEE code of the town, not the postal code. You can find the code on the [INSEE website](https://www.insee.fr/fr/recherche/recherche-geographique).
+
+<p class='note warning'>
+More than 75% of metropolitan France is covered by the service.<br/>
+You can check if your city is covered on the [Météo-France website](http://www.meteofrance.com/previsions-meteo-france/previsions-pluie)
+</p>
+
+The sensor value is the time to next rain, from 0 to 55min.
+If no rain is forecasted for the nex hour, value will be "No rain".
```
Comment (frenck) [inline source/_components/sensor.meteofrance.markdown]:
```suggestion
If no rain is forecasted for the next hour, the value will be "No rain".
```

File: source/_components/sensor.meteofrance.markdown [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+
+Attributes also gives the forecast for the next hour in 5min intervals.
```
Comment (frenck) [inline source/_components/sensor.meteofrance.markdown]:
```suggestion
Attributes also give the forecast for the next hour in 5min intervals.
```

**Review by frenck — CHANGES_REQUESTED** [review]
@victorcerutti Thanks for the PR! I've reviewed it and left you some comments.
Could you please take a look? 👍

**Comment by victorcerutti** [comment]
Thank you for the review.
I have work to do on the sensor and I might change some of the doc.
I take account of the changes requested and will update as soon as the work is done.

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by fabaff — APPROVED** [review]
Thanks :bird:

**Pull request merged**